modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -23,5 +23,5 @@ For more information, see https://oauth.net/2/[OAuth 2.0] and https://openid.net
 You can use the Connected Apps feature to create a seamless authentication experience for end users. Connected apps address use cases for the following types of users:
 
 * An xref:connected-apps-org-admin.adoc[organization administrator] can control how their organization’s data is used by whitelisting apps, revoking access, and disabling this feature for the entire organization.
-* A xref:connected-apps-developers.adoc#developers[developer] can register new, and manage existing, apps within their organization.
+* An xref:connected-apps-developers.adoc#developers[organization administrator who is developing a connected app] can register new, and manage existing, apps within their organization.
```

Comment:
Just because it might be easier to parse...:
```suggestion
* An xref:connected-apps-developers.adoc#developers[organization administrator who is developing a connected app] can register new (and manage existing) apps within their organization.
```